### DISCUSSION TITLE
How to get merged OpenAPI specs after mounting two sub-apps?

### DISCUSSION THREAD

**Discussion by HarshTrivedi** [fastapi/fastapi#11088] [Questions] [answered]
### First Check

- [X] I added a very descriptive title here.
- [X] I used the GitHub search to find a similar question and didn't find it.
- [X] I searched the FastAPI documentation, with the integrated search.
- [X] I already searched in Google "How to X in FastAPI" and didn't find any information.
- [X] I already read and followed all the tutorial in the docs and didn't find an answer.
- [X] I already checked if it is not related to FastAPI but to [Pydantic](https://github.com/pydantic/pydantic).
- [X] I already checked if it is not related to FastAPI but to [Swagger UI](https://github.com/swagger-api/swagger-ui).
- [X] I already checked if it is not related to FastAPI but to [ReDoc](https://github.com/Redocly/redoc).

### Commit to Help

- [X] I commit to help with one of those options 👆

### Example Code

```python
from fastapi import FastAPI


app_one = FastAPI()
app_two = FastAPI()

@app_one.get("/one_endpoint/")
async def one_endpoint():
    return {"source": "App One"}

@app_two.get("/two_endpoint/")
async def two_endpoint():
    return {"source": "App Two"}

main_app = FastAPI()
main_app.mount("/one", app_one)
main_app.mount("/two", app_two)

combined_openapi_schema = main_app.openapi()
print(combined_openapi_schema)

# {'openapi': '3.1.0', 'info': {'title': 'FastAPI', 'version': '0.1.0'}, 'paths': {}}

# ^ I want a combined openapi schema of the two apps here.
```

### Description

I have mounted two fastapi apps into a single fastapi apps with a route prefix for each of them. When I call `main_app.openapi()`, I expect to get the openapi specs of the entire app, which includes the two mounted apps. How can I achieve this?

### Operating System

macOS

### FastAPI Version

0.99.1

### Pydantic Version

1.10.12

### Python Version

Python 3.11.0

**Comment by jgould22** [comment, marked as answer]
The mounted applications are "independent" and each have their own docs accessible at `/one/docs/` and `/two/docs`/ 

https://fastapi.tiangolo.com/advanced/sub-applications/#mounting-a-fastapi-application

You will have to merge the two json together manually if you want the openapi spec for both.

**Comment by shurshilov** [comment]
Hi, i solved issue like this
![image](https://github.com/tiangolo/fastapi/assets/11828278/5f67d99f-d00f-42f3-8948-c8d6577b77ba)
